nautobot/extras/filters.py

```diff
@@ -976,7 +997,9 @@ def search(self, queryset, name, value):
         if not value.strip():
             return queryset
         return queryset.filter(
-            Q(name__icontains=value) | Q(slug__icontains=value) | Q(content_types__model__icontains=value)
+            Q(name__icontains=value)
+            | Q(slug__icontains=value)
+            | Q(content_types__model__icontains=value) | Q(id__iexact=value)
```

Comment:
It may not be over the line length but we should be consistent to move `| Q(id__iexact=value)` to it's own line.

Reply:
I thought I caught it, but it should be fixed with new push